packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -56,6 +55,10 @@ export async function setupWorkspaceOrgType(defaultUsernameOrAlias?: string) {
           setDefaultUsernameHasChangeTracking(false);
           setDefaultUsernameHasNoChangeTracking(false);
           break;
+        case 'NoUsernameFoundError':
```

Comment:
Saw this error when I was writing the unit test.  

<img width="500" alt="2023-04-05_22-35-59" src="https://user-images.githubusercontent.com/46458081/230285896-96fbde14-5fb5-411f-a355-173eb416df8a.png">